extensions/amp-ooyala-player/amp-ooyala-player.md

```diff
@@ -0,0 +1,76 @@
+<!---
+Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# <a name="amp-ooyala-player"></a> `amp-ooyala-player`
+
+<table>
+  <tr>
+    <td width="40%"><strong>Description</strong></td>
+    <td>Displays an <a href="https://www.ooyala.com/">Ooyala</a> video.</td>
+  </tr>
+  <tr>
+    <td width="40%"><strong>Availability</strong></td>
+    <td>Stable</td>
+  </tr>
+  <tr>
+    <td width="40%"><strong>Required Script</strong></td>
+    <td><code>&lt;script async custom-element="amp-ooyala-player" src="https://cdn.ampproject.org/v0/amp-ooyala-player-0.1.js">&lt;/script></code></td>
+  </tr>
+  <tr>
+    <td class="col-fourty"><strong><a href="https://www.ampproject.org/docs/guides/responsive/control_layout.html">Supported Layouts</a></strong></td>
+    <td>FILL, FIXED, FLEX_ITEM, RESPONSIVE</td>
+  </tr>
+  <tr>
+    <td width="40%"><strong>Examples</strong></td>
+    <td></td>
+  </tr>
+</table>
+
+## Example
+
+```html
+<amp-ooyala-player
+    data-embedcode="Vxc2k0MDE6Y_C7J5podo3UDxlFxGaZrQ"
+    data-pcode="5zb2wxOlZcNCe_HVT3a6cawW298X"
+    data-playerid="6440813504804d76ba35c8c787a4b33c"
+    width="640" height="360"></amp-ooyala-player>
+```
+
+## Attributes
+
```

Comment:
OK, clear.  Is there anything else that has to be implemented / reworked/ fixed in this pull request?